twilio/base/client_base.py

```diff
@@ -48,9 +52,8 @@ def __init__(
         """ :type : str """
         self.user_agent_extensions = user_agent_extensions or []
         """ :type : list[str] """
-
```

Comment:
Just test once that since we have removed this, the requests that use accountsid-authToken are being validated correctly for null credentials

Reply:
yes, tested that scenario